prow/github/report/report.go

```diff
@@ -258,10 +260,21 @@ func parseIssueComments(pj prowapi.ProwJob, isBot func(string) bool, ics []githu
 }
 
 func createEntry(pj prowapi.ProwJob) string {
+	required := strconv.FormatBool(true)
```

Comment:
Any job triggered prior to this change will not have this label, and will be labeled required. Should we default to `unknown` instead of true to be more accurate?

Reply:
done